docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -0,0 +1,143 @@
+
+Here we provide a reference of its contents; use it as a reminder when working with the CocoaPods integration.
+
+* Learn how to [set up the environment](native-cocoapods.md#set-up-the-environment-to-work-with-cocoapods) and
+[configure Kotlin Cocoapods Gradle plugin](native-cocoapods.md#add-and-configure-kotlin-cocoapods-gradle-plugin).
+* Depending on your project and purposes, you can add dependencies between [a Kotlin project and a Pod library](native-cocoapods-libraries.md)
+as well as [a Kotlin Gradle project and an Xcode project](native-cocoapods-xcode.md).
+
+
+## ID and version
+
+To apply the CocoaPods plugin, add the following lines to the `build.gradle(.kts)` file:
+
+```kotlin
+plugins {
+   kotlin("multiplatform") version "%kotlinVersion%"
+   kotlin("native.cocoapods") version "%kotlinVersion%"
+}
+```
+
+The plugin versions match the Kotlin release versions. The most recent version is %kotlinVersion%.
+
+## The `cocoapods` block
+
+The `cocoapods` block is the top-level block for the Cocoapods configuration. Inside it, you can use the following blocks,
+functions and properties:
+
+| **Name**                              | **Description**                                                                                                                                                                                                               | 
+|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `version`                             | The version of the pod. If not specified, a Gradle project version is used. If none of these properties are configured, you'll get an error.                                                                                  |
+| `summary`                             | A description of the pod built from this project (mandatory).                                                                                                                                                                 |
+| `homepage`                            | A link to the homepage of the pod built from this project (mandatory).                                                                                                                                                        |
+| `authors`                             | Specifies authors of the pod built from this project.                                                                                                                                                                         |
+| `podfile`                             | Configures the existing pod file.                                                                                                                                                                                             |
```

Comment:
Isn't it a **Podfile**?
https://guides.cocoapods.org/using/the-podfile.html

Reply:
Yes, it should be Podfile, fixed throughout the documentation